### ISSUE TITLE
🐛Search bar does not seem to be finding appropriate results

### ISSUE THREAD

**Issue by laritakr** [notch8/utk-hyku#423]
# Summary
Some (but not all) faceted terms (i.e. keyword) are found, and title words are found. Otherwise the search reports no results.

<details>

![Screenshot 2023-06-07 at 1 32 19 PM](https://github.com/scientist-softserv/utk-hyku/assets/17851674/1de3231c-8424-4314-9ab8-e57d3d739259)
![Screenshot 2023-06-07 at 1 32 40 PM](https://github.com/scientist-softserv/utk-hyku/assets/17851674/932b685f-dc24-4a2a-a8b7-47ec0166667a)
![Screenshot 2023-06-07 at 1 33 40 PM](https://github.com/scientist-softserv/utk-hyku/assets/17851674/8a0c24db-7b33-4062-ac63-db15be2a876a)

</details>

# Accepted Criteria
- [ ] Additional testing will be needed regarding iiif OCR text search, [once derivatives are working](https://github.com/scientist-softserv/utk-hyku/issues/422).
- [ ] Parent work is appropriately returning the child work for the child's title. Other searches do not find the child and return a parent because of the overall search issues.
- [ ] Make sure all of the properties listed as faceted in the metadata profile, appear as facets in the catalog search page.

# Testing Instructions
Make a Generic Work. Add values to the required fields and a file. Add a value to Form Local and save. 
Search the catalog for the value of Form Local. 
- [ ] The work you just created should be returned in the search results. 

Search the catalog for OCR content as well to make sure it's still working

**Comment by ShanaLMoore** [comment]
assumption: if something if faceted we should be able to search for it. cc @laritakr 

STEP 1: How does default Hyku behave?

Jill to create a spike ticket for step 1

**Comment by jillpe** [comment]
blocked by #447

**Comment by ShanaLMoore** [comment]
If AllinsonFlex isn't capable of doing this, manually add them for now and create a separate ticket for AllinsonFlex to do it automatically.

**Comment by DeonFranklin** [comment]
Catalog search is also being handled by: 

- #460

**Comment by jillpe** [comment]
SoftServ QA: Pass!

[Work URL](https://dc.utk-hyku-staging.notch8.cloud/concern/generic_works/256bf94b-e2c7-4890-a6f1-4a2542e37276?locale=en)

Form Local: `#423 QA`

Searched `#423 QA` and it appears in the search results list

<details><summary>Screenshot</summary>
![Image](https://github.com/scientist-softserv/utk-hyku/assets/84697174/b450b9f0-e745-4a19-9cd0-4eb14bd2bc6f)
</details>

Searched `stare` and `minute` and this work appears in the search results list as well (see screenshots below)

<details><summary>Screenshots</summary>


![Image](https://github.com/scientist-softserv/utk-hyku/assets/84697174/1482412b-535d-4e08-9dcc-3491cc046ea1)



![Image](https://github.com/scientist-softserv/utk-hyku/assets/84697174/7a4edd3d-4335-4d1a-b3a5-aee02593cc96)


</details>

**Comment by jillpe** [comment]
Verification on production blocked until #358 resolved
